front/temporal/labs/activities.ts

```diff
@@ -228,22 +233,34 @@ export async function processGoogleDriveTranscriptActivity(
     return;
   }
 
-  // TODO: Consider using a content fragment to attach the transcript.
   const convRes = await dustAPI.createConversation({
-    title: null,
+    title: transcriptTitle,
     visibility: "unlisted",
     message: {
-      content: transcriptContent + extraInstructions,
+      content: "IMPORTANT: Answer in HTML format and NOT IN MARKDOWN.",
```

Comment:
This is not the best message: I would probably try to not have any message at all except for the mention of the assistant following our convention

Reply:
We want to make sure the assistant always answers in HTML though, even if the builder hasn't mentioned it (because markdown in emails is ugly).
Where do you think I can put this?

Reply:
Why do you want to have it in HTML and not markdown? I think we want it in markdown or whathever the user requested no?

The user might request a table or some code or whatever.

Reply:
I presume the reason you want HTML is because you want to embed it in the email but I think running these assistants serve a much broader purpose than embedding the email response. It's fine to just have a link in the email that points to the conversation where the experience will be much richer.

Reply:
I'm not sure to agree about this - we're really "dust search" based, but a lot of companies are "Gmail search" based (even if they have Dust, reflexes are hard to lose) - just sending a link to the conversation seems quite dull to me. 
Maybe I could make the Markdown -> HTML conversation using a lib before the send?

Reply:
The main point is that adding such instructions here will be at odd with what the user may want to do.

Happy to incorporate the text of the conversation in the email but let's do it text/plain then and make no assumption here. Let's also make sure to have a link to the conversation at the top of the email before the converstaion text :+1:

Reply:
Removed that instruction and converted to HTML right before sending the email